### ISSUE TITLE
java.lang.IllegalArgumentException: ByteBuffer is not a valid flatbuffer model issue

### ISSUE THREAD

**Issue by remibasaru** [tensorflow/tensorflow#24322]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on a mobile device: Samsung S5
- TensorFlow installed from (source or binary): Binary
- TensorFlow version: 1.12
- Python version: 3.5
- Installed using virtualenv? pip? conda?: conda
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Describe the problem**
I have created my own architecture, on tensorflow and I have used Toco to convert to .tflite, However the android app still throws the following error: "java.lang.IllegalArgumentException: ByteBuffer is not a valid flatbuffer model."

**Provide the exact sequence of commands / steps that you executed before running into the problem**
'''
       AssetFileDescriptor fileDescriptor = activity.getAssets().openFd(MODEL_PATH);
        FileInputStream inputStream = new FileInputStream(fileDescriptor.getFileDescriptor());
        FileChannel fileChannel = inputStream.getChannel();
        long startOffset = fileDescriptor.getStartOffset();
        long declaredLength = fileDescriptor.getDeclaredLength();
        return fileChannel.map(FileChannel.MapMode.READ_ONLY, startOffset, declaredLength);
'''

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by ymodak** [comment]
Apologies for the delay in response. Is this still an issue for you?
Can you please take a look at duplicate #23628? Thanks!

**Comment by NicolasVidal** [comment]
Hi there, I'm trying to convert a Tensorflow 1.12 / Keras model with TFLiteConverter.from_keras_model_file. I've got no issue doing this on Colaboratory (tf 1.12), but when I run my model I encounter the dreaded error : 
"Caused by: java.lang.IllegalArgumentException: ByteBuffer is not a valid flatbuffer model" error. with firebase-ml-model-interpreter:16.2.4

The model structure is the following : 

Input(884)
BatchNormalization()
Reshape(52,17)
LSTM(256, unroll=True, return_sequences=True)
LSTM(256, unroll=True)
BatchNormalization()
Dense(260)

I previously successfully converted and used similar models, but without Reshape and (unrolled) LSTM layers.

Any advice on version or changes I should make ?

Thank you very much in advance.

**Comment by ymodak** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
@NicolasVidal Can you please post a [new issue](https://github.com/tensorflow/tensorflow/issues/new/choose) explaining your problem if it still persists? thanks

**Comment by alice17** [comment]
I'm having the same issue, converted the model with from_keras_model_file, tried to load it in Android but gives me this error:

`java.lang.IllegalArgumentException: Contents of /conv.tflite does not encode a valid TensorFlowLite model: Could not open '/conv.tflite'.The model is not a valid Flatbuffer file`
